lib/features/context-pad/ContextPad.js

```diff
@@ -422,18 +405,35 @@ ContextPad.prototype.getPad = function(target) {
     event.stopPropagation();
   });
 
-  var activeRootElement = this._canvas.getRootElement();
-
-  this._overlayId = overlays.add(activeRootElement, 'context-pad', overlaysConfig);
-
-  var pad = overlays.get(this._overlayId);
+  this._container.appendChild(html);
 
   this._eventBus.fire('contextPad.create', {
     target: target,
-    pad: pad
+    pad: html
   });
 
-  return pad;
+  return html;
+};
+
+/**
+ * @param {ContextPadTarget} target
+ *
+ * @return { { html: HTMLElement } }
+ */
+ContextPad.prototype.getPad = function(target) {
+  console.warn(new Error(
+    'getPad is deprecated and will be removed in future library versions'
```

Comment:
What is this going to be replaced with? Or is there no use-case to use it?

Reply:
We discussed this today and weren't sure if something like `getHtml` would make sense or whether we should simply ask people to query for the HTML (`canvas.getContainer().querySelector('.djs-context-pad')`.

Reply:
If we deprecate it, we should mention a migration path in the message, either by referencing a different method or linking further documentation. Cf. our `di` error message in bpmn-js: https://github.com/bpmn-io/bpmn-js/blob/b7905948700d8f5c4f42376230500bb385e1ec8d/lib/util/CompatibilityUtil.js#L12